### PR TITLE
Info screen refurb

### LINKED ISSUE #41: Update Q&A page
As user
I want to see a pretty Q&A page.

**Requirements:**

1. Updated content (Marek)
2. Updated app colors for the trophy image
3. Updated text styling

### PR DESCRIPTION
**Pull request by knutm92**
fixes #41

### CODE REVIEW

File: lib/widgets/achievementTracker_widget.dart [hunk 1]
```diff
@@ -27,50 +27,55 @@ class _AchievementTrackerState extends State<AchievementTracker> {
...
-              horizontal: 16.0,
-              vertical: 8.0,
-            ),
-            child: Column(
+    return ExpansionTile(
+      iconColor: AppColors.accentSelect,
+                  textColor: AppColors.accentSelect,
+        title: Text("Osiągnięcia",
+            style: TextStyle(
+                color: AppColors.primaryDark, fontWeight: FontWeight.bold)),
```
Comment (darowsluk) [inline lib/widgets/achievementTracker_widget.dart:35]: Consider using consistent styles, ex.:
Theme.of(context).textTheme.headline4

File: lib/widgets/achievementTracker_widget.dart [hunk 2]
```diff
@@ -27,50 +27,55 @@ class _AchievementTrackerState extends State<AchievementTracker> {
...
+        subtitle: Text(
+            "Sprawdź ile zdobyłeś punktów za dobre odpowiedzi w przygodach."),
```
Comment (darowsluk) [inline lib/widgets/achievementTracker_widget.dart:37]: Consider using text style

File: lib/widgets/infoView_widget.dart [hunk 3]
```diff
@@ -30,15 +33,20 @@ class InfoView extends StatelessWidget {
                     style: Theme.of(context).textTheme.headline3,
                   )),
               ExpansionTile(
-                  title: Text("Gdzie znajdę najbliższą Mszę Świętą?",
+                  iconColor: AppColors.accentSelect,
+                  textColor: AppColors.accentSelect,
+                  leading: Icon(LineIcons.cross),
+                  title: Text("Najbliższa msza święta",
                       style: TextStyle(
```
Comment (darowsluk) [inline lib/widgets/infoView_widget.dart:40]: Consider using consistent styles

File: lib/widgets/infoView_widget.dart [hunk 4]
```diff
@@ -30,15 +33,20 @@ class InfoView extends StatelessWidget {
...
-                          color: AppColors.primaryNormal,
+                          color: AppColors.primaryDark,
                           fontWeight: FontWeight.bold)),
+                  subtitle: Text("Gdzie i w jakich godzinach?"),
```
Comment (darowsluk) [inline lib/widgets/infoView_widget.dart:43]: Consider using consistent styles (perhaps refactoring Text function is the way to go)

File: lib/widgets/infoView_widget.dart [hunk 5]
```diff
@@ -55,16 +63,32 @@ class InfoView extends StatelessWidget {
...
-                                          "Ndz: 8:30, 11:30, 17:00, 17:30",
+                                        Row(
+                                          crossAxisAlignment:
+                                              CrossAxisAlignment.start,
+                                          children: [
+                                            Column(
+                                              crossAxisAlignment:
+                                                  CrossAxisAlignment.start,
+                                              children: [
+                                                Text("Pn-Pt:"),
```
Comment (darowsluk) [inline lib/widgets/infoView_widget.dart:74]: Style

File: lib/widgets/infoView_widget.dart [hunk 6]
```diff
@@ -55,16 +63,32 @@ class InfoView extends StatelessWidget {
...
+                                                Text("Sob:"),
+                                                Text("Ndz:")
+                                              ],
+                                            ),
+                                            Column(
+                                              crossAxisAlignment:
+                                                  CrossAxisAlignment.start,
+                                              children: [
+                                                Text("18:00"),
```
Comment (darowsluk) [inline lib/widgets/infoView_widget.dart:83]: Style

File: lib/widgets/infoView_widget.dart [hunk 7]
```diff
@@ -55,16 +63,32 @@ class InfoView extends StatelessWidget {
...
+                                                Text("8:00, 18:00"),
+                                                Text("8:30, 11:30, 17:30")
+                                              ],
+                                            )
+                                          ],
                                         ),
                                         SizedBox(height: 10),
```
Comment (darowsluk) [inline lib/widgets/infoView_widget.dart:90]: Consider using custom Constant

File: lib/widgets/infoView_widget.dart [hunk 8]
```diff
@@ -80,29 +104,120 @@ class InfoView extends StatelessWidget {
                     )
                   ]),
               ExpansionTile(
-                  title: Text(
-                      "Ilu zakonników mieszka na terenie Klasztoru w Bagnie?",
+                  iconColor: AppColors.accentSelect,
+                  textColor: AppColors.accentSelect,
+                  leading: Icon(LineIcons.users),
+                  title: Text("Zakonnicy w klasztorze",
```
Comment (darowsluk) [inline lib/widgets/infoView_widget.dart:110]: Consider using consistent style

File: lib/widgets/infoView_widget.dart [hunk 9]
```diff
@@ -80,29 +104,120 @@ class InfoView extends StatelessWidget {
...
+                      style: TextStyle(
+                          color: AppColors.primaryDark,
+                          fontWeight: FontWeight.bold)),
+                  subtitle: Text("Ilu ich tutaj mieszka?"),
```
Comment (darowsluk) [inline lib/widgets/infoView_widget.dart:114]: Style

File: lib/widgets/infoView_widget.dart [hunk 10]
```diff
@@ -80,29 +104,120 @@ class InfoView extends StatelessWidget {
...
+                  children: [
+                    Container(
+                      margin: EdgeInsets.symmetric(
+                          horizontal: Constants.insideMargin),
+                      child: Column(
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Text(
```
Comment (darowsluk) [inline lib/widgets/infoView_widget.dart:122]: Style

File: pubspec.yaml [hunk 11]
```diff
@@ -25,12 +25,12 @@ dependencies:
   #cloud_firestore: ^3.1.5 TODO: turn back on once needed
   hive: ^2.0.5
   hive_flutter: ^1.1.0
-  expansion_tile_card: ^2.0.0
   flutter:
     sdk: flutter
   get:
   line_icons: ^2.0.1
   cupertino_icons: ^1.0.2
+  url_launcher: ^6.0.20
```
Comment (darowsluk) [inline pubspec.yaml:33]: Do we need this?

**Review by darowsluk — CHANGES_REQUESTED** [review]
Good work! I think the view looks much better, especially with the icons.
In general I only suggested to use the Text() function with consistent styles. Since it is used often, perhaps it is better to create a custom MyText() with cleaner parameters.

**Comment by knutm92** [comment]
Thank you for the review, I will implement the suggestions in the coming days.

File: pubspec.yaml (hunk 11 above)
Comment (knutm92) [reply pubspec.yaml:33]: Yes, this one is used to activate www.totu.sds.pl link

**Comment by knutm92** [comment]
@darowsluk I refactored the code a bit, mainly added wrappers for Text. I'm not entirely convinced by the way I created them so further suggestions are welcome.
p.s. it seems that the group photo needs some adjustments so I will add it via another PR once ready.

**Comment by darowsluk** [comment]
@knutm92 Here are my suggestions to the text widget wrappers:
1. File text_styles.dart was generated by Supernova from Figma file, so I think it is best not to change its contents manually. I suggest creating a seperate file with just the KompasText wrapper class.
2. I recommend making just one KompasText() class with a parameter called 'style', which will take an enum to select a style from AppTextStyles, namely: headerH1-6, paragraphText, etc.
3. With this mechanism in place we can create a separate issue to refactor all Text() calls into KompasText(). This will enforce consistent text styles throughout the entire app.

**Comment by knutm92** [comment]
@darowsluk 
Good ideas, I have pushed the changes and created a refactoring issue, please have a look.

**Comment by darowsluk** [comment]
The refactoring of KompasText() looks great.
One problem that I had was with the name of two imports:
`import 'package:kompas/statics/kompasTextWrapper.dart';`
After I changed it to "kompasText.dart" everything builds fine.

**Comment by knutm92** [comment]
Repaired :)

**Comment by darowsluk** [comment]
Now everything is OK. :)

**Comment by knutm92** [comment]
Merging is blocked, could you send a review with an approval?

**Review by darowsluk — APPROVED** [review]
Great job. Ready to merge.

**Pull request merged**